pybandits/model.py

```diff
@@ -292,7 +328,9 @@ def _stable_sigmoid(cls, x: Union[np.ndarray, TensorVariable]) -> Union[np.ndarr
             Sigmoid function applied to the input values.
         """
         backend = np if isinstance(x, np.ndarray) else pmath
-        prob = backend.where(x >= 0, 1 / (1 + backend.exp(-x)), backend.exp(x) / (1 + backend.exp(x)))
+        with warnings.catch_warnings():
```

Comment:
why is this catch_warnings needed?

Reply:
exp(...) causes overflow with big numbers